lib/Field/RepeatableSettings.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import { bool, number } from 'prop-types';
+import { ButtonIcon } from '../ButtonNew/ButtonIcon/ButtonIcon';
+import { ButtonIconDanger } from '../ButtonNew/ButtonIcon/ButtonIconDanger';
+
+const RepeatableSettings = ({
+  isTextDirLTR,
+  fieldIsActive,
+  repeatPosition,
+  isLastRepeat
+}) => {
+  const label = `${repeatPosition < 9 ? '0' : ''}${repeatPosition + 1}`;
+
+  return (
+    <div
+      className={`
+      text-neutral-primary
+      text-lg
+      leading-snug
+      flex
+      items-center
+      -my-3
+      ${isTextDirLTR ? 'ml-auto' : 'mr-auto'}
+    `}
+    >
+      <div
+        className={`
+        ${fieldIsActive ? '' : 'opacity-0'}
+        group-hover:opacity-100
+        transition-opacity
+        duration-200
+        ease-animation-curve
```

Comment:
Added our own custom animation curve to match what we were already using for other parts, including the `FieldActions`